packages/rest/src/server.ts

```diff
@@ -47,6 +48,10 @@ export const setupServer = async (agent: Agent, config: ServerConfig) => {
 
   app.use('/docs', swaggerUiExpress.serve, swaggerUiExpress.setup(spec))
 
+  config.staticResources?.map((item) => {
+    app.use(item.pathName, stc(item.directory))
+  })
```

Comment:
I'm thinking whether it may be more flexible to let the user pass an optional `app` instance to the `setupServer` method. As now for every thing the user wants to add we need to add a new option to this package. If we just allow an app the user could modify however he wants and we just add the controllers and `/docs` endpoints.